regolith/schemas.py

```diff
@@ -21,6 +21,8 @@
 GRANT_STATI = ["pending", "declined", "accepted", "in-prep"]
 MILESTONE_TYPES = ["mergedpr", "meeting", "other", "paper", "release", "email", "handin",
                   "approval", "presentation", "report", "submission", "decision", "demo", "skel"]
+NOT_SEMINAR_TYPES = ["award", "contributed_oral", "invited", "keynote",
```

Comment:
this should be handled in the same was as `PROJECTUM_STATI`, so `PRESENTATION_SEMINAR_TYPES` and `PRESENTATION_NON_SEMINAR_TYPES` and then make `PRESENTATION_TYPES` = `PRESENTATION_SEMINAR_TYPES` + `PRESENTATION_NON_SEMINAR_TYPES`